plugins/export_spec_3d_csv.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+'''
```

Comment:
According to PEP these should actually also be triple double quotes. https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring

It looks like most plugins in odemis have triple single quotes, so you can also leave it like this.

Reply:
I agree totally. Actually I think that strings should just always use the double quotes. For consistency I'd like to have no string with single quote. This actually comes from a copy-paste of another plugin. I'll fix this one at least....